cumulusci/cli/cli.py

```diff
@@ -305,115 +317,153 @@ def service(config):
+
+    git_prefix_beta = click.prompt(click.style('Beta Tag Prefix', bold=True), default='beta/')
     if git_prefix_beta and git_prefix_beta != config.global_config.project__git__prefix_beta:
         git_config.append('        prefix_beta: {}'.format(git_prefix_beta))
+
+    git_prefix_release = click.prompt(click.style('Release Tag Prefix', bold=True), default='release/')
     if git_prefix_release and git_prefix_release != config.global_config.project__git__prefix_release:
         git_config.append(
             '        prefix_release: {}'.format(git_prefix_release))
+
     if git_config:
         yml_config.append('    git:')
         yml_config.extend(git_config)
 
     #     test:
     test_config = []
+    click.echo()
+    click.echo(click.style('# Apex Tests Configuration', bold=True, fg='blue'))
+    click.echo('The CumulusCI Apex test runner uses a SOQL where clause to select which tests to run.  Enter the SOQL pattern to use to match test class names.')
+    
+    test_name_match = click.prompt(click.style('Test Name Match', bold=True), default=config.global_config.project__test__name_match)
     if test_name_match and test_name_match != config.global_config.project__test__name_match:
         test_config.append('        name_match: {}'.format(test_name_match))
     if test_config:
         yml_config.append('    test:')
         yml_config.extend(test_config)
 
+    if package_namespace:
+        yam_config.append('orgs:')
+        yam_config.append('    scratch:')
+        yam_config.append('        dev_namespaced:')
+        yam_config.append('            config_file: orgs/dev.json')
+        yam_config.append('            namespaced: True')
+
     yml_config.append('')
 
     with open('cumulusci.yml', 'w') as f_yml:
         f_yml.write('\n'.join(yml_config))
 
-    click.echo("Your project is now initialized for use with CumulusCI")
-    click.echo(
-        "You can use the project edit command to edit the project's config file")
+    click.echo(click.style("Your project is now initialized for use with CumulusCI", bold=True, fg='green'))
+    click.echo(click.style(
+        "You can use the project edit command to edit the project's config file", fg='yellow'
+    ))
 
+    if os.path.isfile('sfdx-project.json'):
+        return
+
+    if not os.path.isdir('src'):
+        os.mkdir('src')
+
+    sfdx_project = {
+        "packageDirectories": [
+            {
+                "path": "src",
+                "default": True,
+            }
+        ],
+        "namespace": package_namespace,
+        "sourceApiVersion": package_api_version,
+}
+    with open('sfdx-project.json','w') as f:
+        f.write(json.dumps(sfdx_project))
+
+    if not os.path.isdir('orgs'):
+        os.mkdir('orgs')
+      
+    org_content_url = 'https://raw.githubusercontent.com/SalesforceFoundation/sfdo-package-cookiecutter/master/%7B%7Bcookiecutter.project_name%7D%7D/orgs/{}.json' 
```

Comment:
this...worries me...

I know it was simple but maybe lets just include these files in cci distribution?

Reply:
Probably something we should do eventually but can be done in a future commit.  This definitely works for now at least.